example/flutter_example/lib/main.dart

```diff
@@ -22,7 +22,31 @@ class HomePage extends StatefulWidget {
 }
 
 class _HomePageState extends State<HomePage> {
-  Ros ros = Ros(url: 'ws://10.0.2.2:9090');
+  Ros ros;
+  Topic chatter;
+
+  @override
+  void initState() {
+    ros = Ros(url: 'ws://10.0.2.2:9090');
+    chatter = Topic(
+        ros: ros, name: '/chatter', type: "std_msgs/String", reconnectOnClose: true, queueLength: 10, queueSize: 10);
+    this.initConnection();
```

Comment:
Don't you want to wait for the user to press the button to initialize the connection?

Reply:
While testing it I didn't want to have to press the button every time. I'll get rid of it now.